prow/initupload/run.go

```diff
@@ -63,22 +67,24 @@ func (o Options) Run() error {
 		return fmt.Errorf("could not resolve job spec: %v", err)
 	}
 
-	started := specToStarted(spec)
+	uploadTargets := map[string]gcs.UploadFunc{}
+
+	failed := false
```

Comment:
nit: `var failed bool`

Reply:
humm sure, is that recommended from go code style?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/initupload/run.go#L117-L123

also that confused me a bit... :joy:

Reply:
done

Reply:
Joe told me that when you want it to default to the default value, use `var foo int` rather than `foo := 0`, whereas you'd do `foo := 1`